waspc/data/Generator/templates/sdk/wasp/env/index.ts

```diff
@@ -0,0 +1,29 @@
+import * as z from 'zod'
+
+const redColor = '\x1b[31m'
+
+export function ensureEnvSchema<Schema extends z.ZodTypeAny>(
+  data: unknown,
+  schema: Schema
+): z.infer<Schema> {
+  try {
+    return schema.parse(data)
+  } catch (e) {
+    if (e instanceof z.ZodError) {
+      const errorOutput = [
+        '',
+        '|══ Env vars validation failed ══',
+        '|',
+      ]
+      for (const error of e.errors) {
+         errorOutput.push(`| - ${error.message}`)
+      }
+      errorOutput.push('|')
+      errorOutput.push('|════════════════════════════════')
+      console.error(redColor, errorOutput.join('\n'))
+      throw new Error('Error parsing environment variables')
+    } else {
+      throw e
+    }
+  }
```

Comment:
I like how you made this very visible, but you having to do that is a nice reminder that our output is very noisy. I hope we clean that up soon